core/templates/components/skill-selector/skill-selector.component.spec.ts

```diff
@@ -102,6 +102,34 @@ describe('SkillSelectorComponent', () => {
     });
   });
 
+  it('should correctly filter untriaged skill summaries', () => {
```

Comment:
These testes are not comprehensive, please cover the different cases that can arise.

Reply:
> These tests are not comprehensive, please cover the different cases that can arise.

Done, thanks!

Reply:
These are fine tests for the search, but where are the tests that check that the issue has been solved correctly (i.e. existing skills are filtered out)? I.e. that the functionality you've added/modified in this PR works?

If you don't add tests for that then it may break again in future.